### ISSUE TITLE
Can you add <bits/stdc++.h> ? 

### ISSUE THREAD

**Issue by DarkWarrior703** [clangd/clangd#353]
I know it is a GCC template, but it is used in Competitve Programming. When I use it, it appears as an error and every function appears also as an error.

**Comment by sam-mccall** [comment]
This is part of the libstdc++ version of the c++ standard library. clangd doesn't ship the c++ standard library but rather uses one found on your system. (Generally, the same one clang uses).

If you're in Mac or FreeBSD, libc++ is the default and you'll need to install libstdc++ and add `-stdlib=libstdc++` to your compilation database.

If you're on another platform, maybe libstdc++ isn't properly installed. Can you compile things with clang or gcc?

**Comment by DarkWarrior703** [comment]
I can compile clang ans also gcc. But it says that that the library is unsupported on clangd

**Comment by sam-mccall** [comment]
I've verified that this header *can* work with clangd when libstdc++ is correctly installed (it works out-of-the-box on a fairly standard debian system).

There's not really anything to be done on the clangd side here (we're unlikely to get involved in distributing standard library, and if we did it would be libc++ which does not include this header). You'd need to provide a lot more information to troubleshoot here, but this isn't a support forum in any case. I do hope you manage to get it working.
